site/content/entry/yuzu-linux-installer/index.md

```diff
@@ -0,0 +1,55 @@
++++
+date = "2022-08-10T23:00:00+05:30"
+title = "New Feature Release - Installer for Linux"
+author = "CaptV0rt3x"
+forum = 612296
++++
+
+Hey there yuz-ers!
+Guess what? We've got some news to share.
+We're excited to announce Linux support for our yuzu installer!
+Let's jump right in!
+
+<!--more-->
+
+## What's happening?
+
+We are very excited to announce Linux compatibility for our yuzu installer.
+The installer will distribute AppImages of mainline and early access builds for our Linux users.
+
+You can now download this installer from our [downloads page](https://yuzu-emu.org/downloads/).
+
+(Note: You need to be a yuzu Patreon Subscriber to get access to the early access builds)
+
+## Why?
+
+When we originally released our installer, our Patreon userbase was predominantly Windows users and truth-be-told, it still is.
+That's why the installer's Linux compatibility wasn't a priority for us then.
+
+{{< imgs
+	"./maintenance.png| As a current limitation, Patreon users will have to paste their token by pressing Ctrl + V"
+  >}}
```

Comment:
```suggestion
```
This image's description text doesn't match the content of the image, and the image itself doesn't add much to the article. Removing